ApiGateway/Program.cs

```diff
@@ -20,9 +26,20 @@
 builder.Services.AddSingleton<Cache<Signal>>();
 builder.Services.AddSingleton<Cache<Tracksection>>();
 
-// TODO: Connection details should be loaded from a config provider
 builder.Services
-    .AddSingleton(static services => new MQTTService(services.GetRequiredService<ILogger<MQTTService>>(), "webapi", "192.168.255.6", 1883, "richard", "password", "signalling/", true, true))
+    .AddSingleton(
```

Comment:
Technically this should use `services.GetRequiredService<IConfiguration>()`, as the builder might not be available at the time